package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "adm-zip": "^0.4.7",
-    "appium-base-driver": "^2.0.3",
+    "appium-base-driver": "^2.0.9",
```

Comment:
It's not a breaking change. It's just something additional that can be made use of here.